pkgs/applications/networking/wgnord/default.nix

```diff
@@ -24,6 +24,7 @@ resholve.mkDerivation rec {
 
   postPatch = ''
     substituteInPlace wgnord \
+      --replace 'conf_dir=/var/lib/wgnord' "conf_dir=$out/share" \
```

Comment:
```
	privkey="$(/nix/store/bpjd8324a6qj34zxhn5asgvi8bpm9bb2-jq-1.7-bin/bin/jq -j '.nordlynx_private_key' $conf_dir/credentials.json)"
```

```
	query "https://$host/v1/users/services/credentials" -H "Content-Type: application/json" -H "Authorization: Basic $auth_token" > $conf_dir/credentials.json
```

```
		longitude="$(/nix/store/bpjd8324a6qj34zxhn5asgvi8bpm9bb2-jq-1.7-bin/bin/jq -er '.longitude' $conf_dir/coords.json)"
		latitude="$(/nix/store/bpjd8324a6qj34zxhn5asgvi8bpm9bb2-jq-1.7-bin/bin/jq -er '.latitude' $conf_dir/coords.json)"
```

`conf_dir` should be kept as is and the template file location changed, that way users can still configure with `/var/lib/wgnord`